clpy/backend/opencl/utility.pyx

```diff
@@ -43,6 +43,19 @@ cdef GetDeviceAddressBits(cl_device_id device):
     ret = valptrs[0]
     return ret
 
+cpdef GetDeviceMaxMemoryAllocation(int device_id):
+    cdef cl_ulong ret
+    cdef size_t actual_size
+    cdef cl_int status = api.clGetDeviceInfo(
+        env.get_devices()[device_id],
+        <cl_device_info>CL_DEVICE_MAX_MEM_ALLOC_SIZE,
+        sizeof(ret),
+        <void *>&ret,
+        &actual_size)
```

Comment:
You can input `NULL` if you won't use actual_size

Reply:
Okay, fixed.